### ISSUE TITLE
who can provide the DET dataset

### ISSUE THREAD

**Issue by liye228** [msracver/Flow-Guided-Feature-Aggregation#66]
thanks you

**Comment by kayser-soze** [comment]
you can download it from [here](http://image-net.org/download-images) after registration

**Comment by Sanku-Yogesh** [comment]
Yes but which dataset exactly could be please refer to this issue https://github.com/Scalsol/mega.pytorch/issues/73#issue-754884469
